src/routes/public/teams.js

```diff
@@ -525,15 +525,15 @@ exports.register = function (server, options, next) {
     config: {
       validate: {
         params: {
-          id: Joi.string().required().description('The team ID')
+          id: Joi.string().required().description('Team ID')
         },
         payload: Joi.object().keys({
-          users: Joi.array().items(Joi.string()).description('User ids')
+          users: Joi.array().items(Joi.string()).description('User IDs')
         }),
         headers
       },
       description: 'Replace team members with the given ones',
-      notes: 'The POST /authorization/teams/{id}/users endpoint replaces all team members',
+      notes: 'The POST /authorization/teams/{id}/users endpoint replaces all team members. The previous ones are removed.',
```

Comment:
Agree, I'll change to `users`. I went with `users` in most places to unify it. Although `members` sounds better.